client/my-sites/email/email-management/email-home.jsx

```diff
@@ -108,11 +111,18 @@ class EmailManagementHome extends Component {
 
 		if ( domainsWithEmail.length < 1 && domainsWithNoEmail.length === 1 ) {
 			return (
-				<EmailProvidersComparisonStacked
-					comparisonContext="email-home-single-domain"
-					selectedDomainName={ domainsWithNoEmail[ 0 ].name }
-					source={ source }
-				/>
+				<>
+					<PageViewTracker
+						path={ emailManagement( ':site', selectedDomainName ?? null ) }
+						title={ 'Email Home' }
+						properties={ { source, context: 'email-home-single-domain' } }
+					/>
+					<EmailProvidersStackedComparison
```

Comment:
Could we add a blank line above so this block of code is easier to read? Thanks!